test/app-luatest/console_debugger_session_test.lua

```diff
@@ -0,0 +1,134 @@
+local t = require('luatest')
+local popen = require('popen')
+local tnt = require('tarantool')
+local g = t.group()
+
+local function normalize_path(s)
+    return s:gsub("^@", ""):gsub("[^/]+$", "")
+end
+
+local function unescape(s)
+    return s:gsub('[\27\155][][()#;?%d]*[A-PRZcf-ntqry=><~]', '')
+end
+
+local function trim(s)
+    return s:gsub('%s+$', '')
+end
+
+local function tarantool_path(arg)
+    local index = -2
+    -- arg[-1] is guaranteed to be non-null
+    while arg[index] do index = index - 1 end
+    return arg[index + 1]
+end
+
+local TARANTOOL_PATH = tarantool_path(arg)
+local path_to_script = normalize_path(debug.getinfo(1, 'S').source)
+local debug_target_script = path_to_script .. 'debug-target.lua'
+
+local DEBUGGER = 'luadebug.lua'
+local dbg_header = DEBUGGER .. ": Loaded for " .. tnt.version
```

Comment:
I'm curious where have it found 'debugger' module for you? Somewhere inside of `.rocks` directory? It's `luadebug` recently, since rename.
```
14:57 $ ./src/tarantool 
Tarantool 2.11.0-entrypoint-538-gda27f2c9c
type 'help' for interactive help
tarantool> dbg = require 'debugger'
---
- error: "[string \"dbg = require 'debugger'\"]:1: module 'debugger' not found:\n\tno
    field package.preload['debugger']\n\tno file '/home/tsafin/datetime/tarantoolt/build/debugger.lua'\n\tno
    file '/home/tsafin/datetime/tarantoolt/build/debugger/init.lua'\n\tno file '/home/tsafin/datetime/tarantoolt/build/debugger.so'\n\tno
    file '/home/tsafin/datetime/tarantoolt/build/.rocks/share/tarantool/debugger.lua'\n\tno
    file '/home/tsafin/datetime/tarantoolt/build/.rocks/share/tarantool/debugger/init.lua'\n\tno
    file '/home/tsafin/datetime/tarantoolt/.rocks/share/tarantool/debugger.lua'\n\tno
    file '/home/tsafin/datetime/tarantoolt/.rocks/share/tarantool/debugger/init.lua'\n\tno
    file '/home/tsafin/datetime/.rocks/share/tarantool/debugger.lua'\n\tno file '/home/tsafin/datetime/.rocks/share/tarantool/debugger/init.lua'\n\tno
    file '/home/tsafin/.rocks/share/tarantool/debugger.lua'\n\tno file '/home/tsafin/.rocks/share/tarantool/debugger/init.lua'\n\tno
    file '/home/.rocks/share/tarantool/debugger.lua'\n\tno file '/home/.rocks/share/tarantool/debugger/init.lua'\n\tno
    file '/.rocks/share/tarantool/debugger.lua'\n\tno file '/.rocks/share/tarantool/debugger/init.lua'\n\tno
    file '/home/tsafin/datetime/tarantoolt/build/.rocks/lib/tarantool/debugger.so'\n\tno
    file '/home/tsafin/datetime/tarantoolt/.rocks/lib/tarantool/debugger.so'\n\tno
    file '/home/tsafin/datetime/.rocks/lib/tarantool/debugger.so'\n\tno file '/home/tsafin/.rocks/lib/tarantool/debugger.so'\n\tno
    file '/home/.rocks/lib/tarantool/debugger.so'\n\tno file '/.rocks/lib/tarantool/debugger.so'\n\tno
    file '/home/tsafin/.luarocks/share/lua/5.1/debugger.lua'\n\tno file '/home/tsafin/.luarocks/share/lua/5.1/debugger/init.lua'\n\tno
    file '/home/tsafin/.luarocks/share/lua/debugger.lua'\n\tno file '/home/tsafin/.luarocks/share/lua/debugger/init.lua'\n\tno
    file '/usr/local/share/tarantool/debugger.lua'\n\tno file '/usr/local/share/tarantool/debugger/init.lua'\n\tno
    file '/usr/share/tarantool/debugger.lua'\n\tno file '/usr/share/tarantool/debugger/init.lua'\n\tno
    file '/usr/local/share/lua/5.1/debugger.lua'\n\tno file '/usr/local/share/lua/5.1/debugger/init.lua'\n\tno
    file '/usr/share/lua/5.1/debugger.lua'\n\tno file '/usr/share/lua/5.1/debugger/init.lua'\n\tno
    file '/home/tsafin/.luarocks/lib/lua/5.1/debugger.so'\n\tno file '/home/tsafin/.luarocks/lib/lua/debugger.so'\n\tno
    file '/usr/local/lib/x86_64-linux-gnu/tarantool/debugger.so'\n\tno file '/usr/local/lib/tarantool/debugger.so'\n\tno
    file '/usr/lib/x86_64-linux-gnu/tarantool/debugger.so'\n\tno file '/usr/local/lib/x86_64-linux-gnu/lua/5.1/debugger.so'\n\tno
    file '/usr/local/lib/lua/5.1/debugger.so'\n\tno file '/usr/lib/x86_64-linux-gnu/lua/5.1/debugger.so'"
...

tarantool> dbg = require 'luadebug'
---
...

tarantool> dbg()
---
- error: 'builtin/luadebug.lua:706: luadebug.lua is not yet compatible with interactive
    Tarantool console'
...
```